themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,56 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
+		(function(initSplitSuffix) {
+
+			initSplitSuffix(window.jQuery);
+
+		}(function($) {
+
+				$(function () {
+					var reg = /^\s*\.?\s?(\d*)\s*(.*)\s*$/,
+						parse = function (val) {
+							if (val && typeof val === 'string') {
+								var parts = val.match(reg);
+								parts.shift();
+								return parts;
+							}
+						};
+					$('#idno_accession_series_and_parts').each(function () {
+						var $dot = $('<span class="idno_dot">.</span>').hide(),
+							$series = $('<input type="number" class="idno_accession_series" title="Enter series number" step="1" min="1"/>'),
+							$suffix = $('<input type="text" class="idno_accession_suffix" size="10"/>'),
+							$hidden = $(this).hide().before($dot).before($series).before($suffix),
+							seriesAndParts = parse($hidden.val()),
+							events = 'keypress change',
+							serialize = function(){
+								var ret = '';
+								if ($series.val()){
```

Comment:
You want to do the `trim()` on this line, too, otherwise the dot will show and be added if the input is whitespace.  Probably put `$series.val().trim()` into a variable and use it here and on line 113.